tests/e2e/util.go

```diff
@@ -1611,6 +1612,66 @@ func getWCPSessionId(hostname string, username string, password string) string {
 
 }
 
+//getk8sWindowsWorkerIPs returns the windows worker node IPs
+func getk8sWindowsWorkerIPs(ctx context.Context, client clientset.Interface, nodeName string) string {
+	var err error
+	nodes, err := client.CoreV1().Nodes().List(ctx, metav1.ListOptions{})
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	var windowsWorkerIp string
+	for _, node := range nodes.Items {
+		if node.Name == nodeName {
+			addrs := node.Status.Addresses
+			for _, addr := range addrs {
+				if addr.Type == v1.NodeInternalIP && (net.ParseIP(addr.Address)).To4() != nil {
+					windowsWorkerIp = addr.Address
+				}
+			}
+		}
+	}
+        gomega.Expect(windowsWorkerIp).NotTo(gomega.BeEmpty(), "Unable to find k8s windows worker IP")
+        return windowsWorkerIp
+	
+} 
+
+//execCommanOnWindowsWorker func returns the size of the pod
```

Comment:
size of pod or volume?

Reply:
its size of the volume and I have updated the same